entry.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash --login
+set -e
+conda activate asf-tools
+exec calculate_hand "$@"
```

Comment:
Did you change the `make_hand` to `calculate_hand`? It's still `make_hand` in `setup.py`:
https://github.com/ASFHyP3/asf-tools/blob/hand-platform/setup.py#L63

Merge conflict resolution problem?